libvast_test/src/table_slices.cpp

```diff
@@ -50,9 +50,9 @@ make_random_table_slices(size_t num_slices, size_t slice_size,
   format::test::reader src{defaults::import::table_slice_type, std::move(opts),
                            nullptr};
   src.schema(std::move(sc));
-  std::vector<table_slice_ptr> result;
-  auto add_slice = [&](table_slice_ptr ptr) {
-    ptr.unshared().offset(offset);
+  std::vector<table_slice> result;
+  auto add_slice = [&](table_slice ptr) {
```

Comment:
`ptr` -> `slice`

Reply:
Good catch. I did `git grep 'table_slice ptr'` and this was the only one I overlooked.